jmx-http/src/main/java/io/airlift/jmx/MBeanResource.java

```diff
@@ -39,13 +39,13 @@
 public class MBeanResource
 {
     private final MBeanServer mbeanServer;
-    private final ObjectMapper objectMapper;
+    private final JsonMapper jsonMapper;
```

Comment:
Same here. I manually changed this to `ObjectMapper` and it still compiles. Consider any case of this and reconsider changing to `JsonMapper`.